api/src/user/mutations/authenticate.js

```diff
@@ -164,8 +166,9 @@ export const authenticate = new mutationWithClientMutationId({
 
       // if user wants to stay logged in create normal http cookie
       if (user.refreshInfo.rememberMe) {
+        const tokenMaxAgeSeconds = jwt.decode(token).exp - jwt.decode(token).iat
         cookieData = {
-          maxAge: 1000 * 60 * 60 * 24 * REFRESH_TOKEN_EXPIRY,
+          maxAge: tokenMaxAgeSeconds,
```

Comment:
refresh tokens maxAge should be kept the same since they are naturally longer lived